package.json

```diff
@@ -13,7 +13,7 @@
   },
   "dependencies": {
     "@calblueprint/prettier-config": "^0.0.1",
-    "@supabase/supabase-js": "^2.36.0",
+    "@supabase/supabase-js": "^2.37.0",
```

Comment:
Was this just an update in supabase? just checking if there was any significant difference here :)

Reply:
This was from when I had to create the auto-generated schema we didn't end up using. I'll revert back to 2.36.0!